dtbase/webapp/app/users/templates/users.html

```diff
@@ -2,7 +2,16 @@
 
 {% block title %} Users {% endblock title %}
 
+{% block stylesheets %}
+  {{ super() }}
+  <!-- <link href="{{ url_for('static', filename='css/create_user.css') }}" rel="stylesheet"> -->
+{% endblock stylesheets %}
+
+
 {% block content %}
+<!-- Utility -->
+<script src="{{url_for('static', filename='javascript/utility.js')}}"></script>
```

Comment:
I have tried to do this, but when I load the page and try to click the button, apparently, the function wasn't loaded. So I am not sure if I need to do some trick to put back to the javascript block and make sure that functions are loaded beforehand?

Reply:
Hmmm. I'll be in meetings from now until 4pm, but I can try to look into this as well. I don't fully understand what the significance of the `{% block javascripts %}` thing is, might have to read a bit more. Could also check how we do these on other pages that have similar features.

Reply:
So apparently, there is a difference between body block and content block (we still don't know what's the difference), so the js block works if added in a body block, or it can be outside if there is a content block